packages/components/src/CollapsiblePanel/CollapsiblePanel.scss

```diff
@@ -141,6 +131,70 @@ $tc-collapsible-panel-btn-color: #555964;
 				flex-basis: 90%;
 				flex-shrink: 0;
 				flex-grow: 1;
+				font-weight: normal;
+			}
+		}
+	}
+}
+
+.descriptive-panel {
+	border-radius: 0;
+	margin-bottom: 0;
+
+	:global(.panel-heading) {
+		background-color: white;
+		border-color: transparent;
+	}
+
+	.group {
+		justify-content: flex-start;
+		display: flex;
+	}
+
+	.title {
+		text-transform: lowercase;
+		color: $tc-collapsible-selectable-panel-title-color;
+		font-weight: bold;
+		overflow: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+	}
+
+	.tag {
+		color: $tc-collapsible-selectable-panel-content-description-color;
+		margin-left: 0;
+		white-space: nowrap;
+		font-size: 12px;
+	}
+
+	.detail {
+		color: $tc-collapsible-selectable-panel-title-color;
+		white-space: nowrap;
+		justify-content: flex-end;
+	}
+
+	:global(.panel-body) {
+		padding: 10px;
+		padding-top: 0;
+
+		.content {
+			display: flex;
+			flex-direction: column;
+
+			.head {
+				display: flex;
+				justify-content: space-between;
+
+				> span {
+					overflow: hidden;
```

Comment:
It caused into a tooltip pointing far from the label in both the right and the left spans (21 steps & user name)